include/vcpkg/build.h

```diff
@@ -220,9 +220,19 @@ namespace vcpkg
 
     StringLiteral to_string_locale_invariant(const BuildResult build_result);
     LocalizedString to_string(const BuildResult build_result);
-    LocalizedString create_user_troubleshooting_message(const InstallPlanAction& action,
-                                                        const VcpkgPaths& paths,
-                                                        Optional<Path>&& issue_body = nullopt);
+    LocalizedString create_user_troubleshooting_message(const InstallPlanAction& action, const VcpkgPaths& paths);
```

Comment:
Since it's now an error to call this IMO it would be better to just inline it (and put the definition of `print_user_troubleshooting_message` into `build.cpp`?)

Reply:
This is not possible:

```
vcpkg-tool\src\vcpkg\install.cpp(525,17): error C3861: "print_user_troubleshooting_message": identifier not found
```